### PR TITLE
Upgrade #1

### PR DESCRIPTION
**Pull request by odyslam**

### CODE REVIEW

**Comment by odyslam** [comment]
Let's rename pending-upgrade to upgrade @anna-carroll

File: development.json [hunk 1]
```diff
@@ -952,23 +951,17 @@
...
-    "arbitrumrinkeby": [
-      "https://rinkeby.arbitrum.io/rpc"
-    ],
     "evmostestnet": [
-      "https://eth.bd.evmos.dev:8545"
+      "https://eth.bd.evmos.dev:8545/"
     ],
     "goerli": [
-      "https://goerli-light.eth.linkpool.io"
+      "https://goerli.infura.io/v3/de08c16049f346fb87575404e21d2b5e"
```
Comment (anna-carroll) [inline development.json:958]: remove

File: development.json [hunk 2]
```diff
@@ -952,23 +951,17 @@
...
     ],
     "neontestnet": [
       "https://proxy.devnet.neonlabs.org/solana"
     ],
-    "optimismkovan": [
-      "https://kovan.optimism.io"
-    ],
     "rinkeby": [
-      "https://rinkeby-light.eth.linkpool.io"
+      "https://eth-rinkeby.alchemyapi.io/v2/-uKtZgu7bWLDEuW3EaOZ0f6eKpqiH-Tj"
```
Comment (anna-carroll) [inline development.json:964]: remove

File: staging.json [hunk 3]
```diff
@@ -596,17 +596,23 @@
...
     ],
     "neontestnet": [
       "https://proxy.devnet.neonlabs.org/solana"
     ],
+    "optimismkovan": [
+      "https://kovan.optimism.io"
+    ],
     "rinkeby": [
-      "https://rinkeby-light.eth.linkpool.io"
+      "https://eth-rinkeby.alchemyapi.io/v2/-uKtZgu7bWLDEuW3EaOZ0f6eKpqiH-Tj"
```
Comment (anna-carroll) [inline staging.json:615]: remove

**Review by anna-carroll — APPROVED** [review]
lgtm

TODO: must upgrade custom token on evmos

**Pull request merged**